<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCloudForm</class>
 <widget class="QWidget" name="QCloudForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_downfile">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>620</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_cloud">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
